### DISCUSSION TITLE
nvim exit insert mode DELAY

### DISCUSSION THREAD

**Discussion by darianmorat** [wezterm/wezterm#6149] [Q&A]
### What Operating System(s) are you running on?

Windows

### Which Wayland compositor or X11 Window manager(s) are you using?

Glaze Window Manager (GlazeWM)
Windows 11

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

When I try to exit nvim (using TMUX) from INSERT mode i got a delay of approximately 600ms... In windows terminal (which is slower than wezterm) I don't get that delay, is almost instant the change from INSERT to NORMAL mode

Edit: 
when I press ESC to exit IINSERT mode and I instantly use j or k to go down or up it just refresh and change the mode... If i don't do that it just takes those almost 500-600ms to show the change to NORMAL mode in the statusbar and also the cursor from Thin to Fat

**Comment by bew** [comment]
Did you try without tmux?

It is known that tmux as a long default escape time, see the various answers to https://vi.stackexchange.com/questions/16148/slow-vim-escape-from-insert-mode
